src/StochasticStyles/spawning.jl

```diff
@@ -1,23 +1,16 @@
 """
-    projected_deposit!(w, add, val, parent, threshold=0, [report_annihilations::Bool])
+    projected_deposit!(w, add, val, parent, threshold=0)
 
 Like [`deposit!`](@ref), but performs threshold projection before spawning. If `eltype(w)`
 is an `Integer`, values are stochastically rounded.
 
-Returns the value deposited and the number of annihilations.
+Returns the value deposited and the number.
```

Comment:
```suggestion
Returns the value deposited.
```

I believe it just returns the value now?